exporter/clickhouseexporter/exporter_traces.go

```diff
@@ -38,6 +38,15 @@ type tracesExporter struct {
 	cfg    *Config
 }
 
+const (
+	otelScopeName    = "otel.scope.name"
+	otelScopeVersion = "otel.scope.version"
+
+	// those are deprecated aliases, but we still report them for backward compatibility reasons
+	otelLibraryName    = "otel.library.name"
+	otelLibraryVersion = "otel.library.version"
```

Comment:
Why is the backward compatibility needed for something gone before even the clickhouse exporter is introduced? I'd suggest not to put redundant fields